exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,8 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+
+var percentStudents = numberOfStudents / (numberOfStudents + numberOfMentors);
+var percentMentors = numberOfMentors / (numberOfMentors + numberOfStudents);
+
+console.log("Percentage students: " + Math.round(percentStudents*100));
+console.log("Percentage mentors: " + Math.round(percentMentors*100));
```

Comment:
The solution looks great, the only thing that you missed is the percentage symbol **+"%"** that you add to your console.log()